firmware/application/transmitter_model.hpp

```diff
@@ -1,6 +1,7 @@
 /*
```

Comment:
Changes to the model files are mostly just re-ordering to that they are easier to navigate (fields, setters, getters all in the same order).

I'll point on the few actual changes in comments.